core/src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -347,6 +347,17 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         return currencies;
     }
 
+    public static List<TradeCurrency> getAllCapitualCurrencies() {
+        List<TradeCurrency> currencies = new ArrayList<>(Arrays.asList(
+                new FiatCurrency("EUR"),
+                new FiatCurrency("BRL"),
+                new FiatCurrency("GBP"),
+                new FiatCurrency("USD")
+        ));
+        currencies.sort(Comparator.comparing(TradeCurrency::getCode));
+        return currencies;
```

Comment:
Sorting is not necessary when you initialize it in the correct order
```suggestion
        return new ArrayList<>(Arrays.asList(
                new FiatCurrency("BRL"),
                new FiatCurrency("EUR"),
                new FiatCurrency("GBP"),
                new FiatCurrency("USD")
        ));
```